client/src/Sidebar.tsx

```diff
@@ -111,7 +111,14 @@ export function Sidebar() {
     <div id="sidebar" className="unselectable" onClick={dispatchUnlessError({ type: "selectCitation" })}>
       <h3 id="citations-label">Review Citations</h3>
       <div className="sidebar-divider" />
-      <div id="docs">
+      <div id="disclaimer-section">
+        <i>
```

Comment:
Please set the style in the .css file, not as an explicit tag.

Reply:
Thanks, updated